components/molecules/AuthSection/auth-section.tsx

```diff
@@ -26,11 +27,20 @@ import { authSession } from "lib/hooks/authSession";
 import { Spinner } from "components/atoms/SpinLoader/spin-loader";
 
 const AuthSection: React.FC = ({}) => {
+  const router = useRouter();
+  const currentPath = router.asPath;
+
   const { signIn, signOut, user, sessionToken } = useSupabaseAuth();
   const { onboarded } = useSession();
   const [notifications, setNotifications] = useState<DbUserNotification[]>([]);
   const [loading, setLoading] = useState(false);
   const [userInfo, setUserInfo] = useState<DbUser | undefined>(undefined);
+  const [host, setHost] = useState<string>("");
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      setHost(window.location.origin as string);
+    }
+  }, []);
```

Comment:
We've done this in a few places now. This should be a reusable hook

```suggestion
  const [host, setHost] = useState<string>("");
  useEffect(() => {
    if (typeof window !== "undefined") {
      setHost(window.location.origin as string);
    }
  }, []);
```